### PR TITLE
Playback queue improvements

### PR DESCRIPTION
**Pull request by lucasnz**
Added option to sort playback queue descending or ascending
Added option "autoPlaylist":
1. automatically selects all episodes in the playback queue menu
2. deletes played episodes

Changed behaviour so it matches the playback queue sort order

### CODE REVIEW

File: app/resources/settings.xml [hunk 1]
```diff
@@ -43,6 +51,18 @@
...
         <setting propertyKey="@Properties.settingServiceUrl" title="@Strings.settingServiceUrlTitle">
             <settingConfig type="alphaNumeric" required="false" />
+        </setting>
+                <setting propertyKey="@Properties.settingPlaylistSortDescending" title="@Strings.settingPlaylistSortDescendingTitle">
+            <settingConfig type="list">
+                <listEntry value="0">@Strings.ascending</listEntry>
+                <listEntry value="1">@Strings.descending</listEntry>
+            </settingConfig>
+        </setting>
+                <setting propertyKey="@Properties.settingPlaylistAutoSelect" title="@Strings.settingPlaylistAutoSelectTitle">
```
Comment (lucasasselli) [inline app/resources/settings.xml:61]: Maybe use a boolean setting instead? 

https://developer.garmin.com/connect-iq/core-topics/app-settings/

Comment (lucasnz) [reply app/resources/settings.xml:61]: Both the new parameters could be booleans. I'm happy to change these both to boolean if you prefer - there's no technical reason. I thought having a drop down list would be clearer to the user. Particularly for the sort order option.
E.g. sort oldest to newest: true/false - False could be interpreted as newest to oldest OR don't sort.

I also wondered about having a separate parameter for delete after played.

I'd totally take your lead on these... But if you prefer booleans, can you suggest the wording to use?

Comment (lucasnz) [reply app/resources/settings.xml]: See latest commit. I have changed settingPlaylistAutoSelect to Boolean and added settingDeleteAfterPlayed.

I'm not sure what to do with settingPlaylistSortDescending. What do you think?

Comment (lucasasselli) [reply app/resources/settings.xml]: I would leave it as it is: drop down menu looks good! Not 100% convinced by the auto select. Does it makes sense to allow the user to manage the queue if that option is true?

**Pull request merged**